frontend_tests/run-casper

```diff
@@ -111,8 +110,14 @@ def run_tests(realms_have_subdomains, files):
             time.sleep(0.1)
         sys.stdout.write('\n')
 
-        print("Running %s" % (cmd,))
-        ret = subprocess.call(cmd, shell=True)
+        for test_file in test_files:
+            cmd = "node_modules/.bin/casperjs %s test --subdomains=%s %s" % (
+                remote_debug, realms_have_subdomains, test_file)
+            print("\n\nRunning %s" % (cmd,))
+            ret = subprocess.call(cmd, shell=True)
+            if ret != 0:
+                break
+            time.sleep(0.3) # give a little breathing room for cleanup
```

Comment:
This 0.3s sleep makes me a bit unhappy, since it seems like the kind of thing that might fail nondeterministically on slow systems.  I tried removing it on my laptop, and the tests passed; maybe it's worth at last adding a block comment explaining a bit more of the story of the debugging process that led to it, so that if we later want to try removing it, we at least know how to test for it?